src/components/page-environment/environment-alerting.js

```diff
@@ -91,24 +91,24 @@ const EnvironmentAlerting = ({
       {alertingConfig && (
         <>
           <Typography>
-            <Icon
-              data={alertingConfig?.enabled ? notifications : notifications_off}
-            ></Icon>
             Alerting is{' '}
             <Typography as="span" bold>
               {alertingConfig?.enabled ? 'enabled' : 'disabled'}
-            </Typography>
-          </Typography>
-          <Typography>
-            <Button variant="outlined" onClick={() => setVisibleScrim(true)}>
-              Configure
+            </Typography>{' '}
+            <Button variant="ghost" onClick={() => setVisibleScrim(true)}>
+              Setup alert
```

Comment:
When alerting is already setup, say something like `Edit alert`?
```suggestion
              {alertingConfig?.enabled ? 'Edit alert' : 'Setup alert'}
```

Reply:
Good suggestion. Fixed